server/routes/temporaryabsences/confirmTemporaryAbsenceAddedToRollController.ts

```diff
@@ -1,9 +1,27 @@
 import { RequestHandler } from 'express'
+import type { PrisonService } from '../../services'
 
 export default class ConfirmTemporaryAbsenceAddedToRollController {
+  public constructor(private readonly prisonService: PrisonService) {}
+
   public view(): RequestHandler {
     return async (req, res) => {
-      return res.render('pages/temporaryabsences/confirmTemporaryAbsenceAddedToRoll.njk', {})
+      const { prisonNumber } = req.params
+      const { activeCaseLoadId } = res.locals.user
+      const { firstName, lastName } = req.flash('prisoner')?.[0] as Record<string, string>
+
+      if (!firstName && !lastName) {
```

Comment:
wouldn't you want to redirect if either of these weren't present?

Reply:
done

Reply:
We added  if (!firstName || !lastName || !prisonNumber || !prison )  instead